### ISSUE TITLE
Dropout operator implementation is not found

### ISSUE THREAD

**Issue by ANSHUMAN87** [dmlc/tvm#814]
I was going through list of operators registered/exposed in TVM, but i am unable to find implementation for Dropout operator, but i can see in NNVM this operator is used, and models are working fin with this operator.
  Can anyone please help me find the implementation code.

**Comment by siju-samuel** [comment]
the current implementation can be seen in SimplifyInference.cc file in nnvm/src/compiler folder
`    } else if (n->op() == dropout_op) {
      NodeEntry undef = MakeNode("__undef__", "undef", {});
      *ret = {n->inputs[0], undef};
      return true;
    } else {
`
But i think nothing is being done as part of this operator as of now because dropout is a regularization technique for reducing overfitting in neural networks by preventing complex co-adaptations on **training data.**

**Comment by tqchen** [comment]
drop out is simply optimized away for inference.
